app/questionnaire/questionnaire_schema.py

```diff
@@ -209,6 +209,12 @@ def get_post_submission(self) -> ImmutableDict:
         schema: ImmutableDict = self.json.get("post_submission", ImmutableDict({}))
         return schema
 
+    @property
+    def is_view_submitted_response_enabled(self) -> bool:
```

Comment:
why is this  `property` instead of method?

Reply:
Why does it need to be a method?. I see it as a property since we do not pass any arguments, which is consistent with our other `is_` functions on questionnaire schema class. It uses method when you have to pass args, and property when you don't.

Reply:
Fair enough ? could it be a `@cached_property` like other properties to be consistent ? i am not sure.

Reply:
Good point, changed to `@cached_property`.